proxy/logging/LogConfig.cc

```diff
@@ -616,15 +616,9 @@ LogConfig::update_space_used()
         //
         // then check if the candidate belongs to any given log type
         //
-        ts::TextView type_name(entry->d_name, strlen(entry->d_name));
-        // A rolled log will look something like:
-        // squid.log_some.hostname.com.20191029.18h15m02s-20191029.18h30m02s.old
-        //
-        // The following logic cuts things back to original unrolled file which
-        // is the key into the deleting_info map (squid.log in the above example).
-        auto suffix = type_name;
-        type_name.remove_suffix(suffix.remove_prefix(suffix.find('.') + 1).remove_prefix(suffix.find('_')).size());
-        auto iter = deleting_info.find(type_name);
+        ts::TextView rolled_filename(entry->d_name, strlen(entry->d_name));
+        auto type_name = LogUtils::get_unrolled_filename(rolled_filename);
```

Comment:
It isn't obvious what the return type of the method is.  We should use a type here.

Reply:
This is resolved by not having the variable name now.